src/Microsoft.AspNet.Identity/IdentityErrorDescriber.cs

```diff
@@ -15,6 +15,15 @@ public virtual IdentityError DefaultError()
                 Description = Resources.DefaultError
             };
         }
+        public virtual IdentityError ConcurrencyFailure()
+        {
+            return new IdentityError
+            {
+                Code = nameof(ConcurrencyFailure),
```

Comment:
Not exactly, they are meant to be logical strings, as opposed to caller, i.e. Logging could print out the error code, instead of the actual error string if desired.  Without the code developers would have to compare against the error strings itself which was a big complaint in the 2.0 stack with identityResult only having an IEnumerable of error strings